e2e/pipelines/test_hybrid_doc_search_pipeline.py

```diff
@@ -47,6 +48,10 @@ def test_hybrid_doc_search_pipeline(tmp_path):
         Document(content="My name is Giorgio and I live in Rome."),
     ]
     hybrid_pipeline.get_component("bm25_retriever").document_store.write_documents(documents)
+    doc_embedder = SentenceTransformersDocumentEmbedder(model="sentence-transformers/all-MiniLM-L6-v2")
```

Comment:
Let's use the same component in the pipeline and here to prevent loading the same model twice.